azure-functions-maven-plugin/src/test/java/com/microsoft/azure/maven/function/handlers/AnnotationHandlerImplTest.java

```diff
@@ -93,7 +93,8 @@ public void cosmosDBTriggerMethod(@CosmosDBTrigger(name = "cosmos",
                                                 databaseName = "db",
                                                 collectionName = "cl",
                                                 connectionStringSetting = "conn",
-                                                leaseCollectionName = "lease") String in) {
+                                                leaseCollectionName = "lease",
+                                                createLeaseCollectionIfNotExists = false) String in) {
```

Comment:
Can you please add test that sets all the properties on cosmos db trigger and verfies generated function.json?